### ISSUE TITLE
Crash report 7.0-rc-2: ISE in WPLegacyMigrationUtils.getDraftsFromDeprecatedDB

### ISSUE THREAD

**Issue by maxme** [wordpress-mobile/WordPress-Android#5451]
```
Fatal Exception: java.lang.RuntimeException: Unable to create application org.wordpress.android.WordPress: java.lang.IllegalStateException: Couldn't read row 0, col -1 from CursorWindow.  Make sure the Cursor is initialized correctly before accessing data from it.
       at android.app.ActivityThread.handleBindApplication(ActivityThread.java:4597)
       at android.app.ActivityThread.access$1600(ActivityThread.java:151)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1414)
       at android.os.Handler.dispatchMessage(Handler.java:110)
       at android.os.Looper.loop(Looper.java:193)
       at android.app.ActivityThread.main(ActivityThread.java:5345)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:515)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:824)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:640)
       at dalvik.system.NativeStart.main(NativeStart.java)
Caused by java.lang.IllegalStateException: Couldn't read row 0, col -1 from CursorWindow.  Make sure the Cursor is initialized correctly before accessing data from it.
       at android.database.CursorWindow.nativeGetLong(CursorWindow.java)
       at android.database.CursorWindow.getLong(CursorWindow.java:507)
       at android.database.AbstractWindowedCursor.getLong(AbstractWindowedCursor.java:86)
       at org.wordpress.android.util.WPLegacyMigrationUtils.getDraftsFromDeprecatedDB(WPLegacyMigrationUtils.java:298)
       at org.wordpress.android.util.WPLegacyMigrationUtils.migrateDraftsFromDeprecatedDB(WPLegacyMigrationUtils.java:80)
       at org.wordpress.android.WordPress.migrateDrafts(WordPress.java:356)
       at org.wordpress.android.WordPress.migrateSelfHostedSites(WordPress.java:350)
       at org.wordpress.android.WordPress.migrateAccessToken(WordPress.java:328)
       at org.wordpress.android.WordPress.runFluxCMigration(WordPress.java:304)
       at org.wordpress.android.WordPress.onCreate(WordPress.java:241)
       at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:1007)
       at android.app.ActivityThread.handleBindApplication(ActivityThread.java:4594)
       at android.app.ActivityThread.access$1600(ActivityThread.java:151)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1414)
       at android.os.Handler.dispatchMessage(Handler.java:110)
       at android.os.Looper.loop(Looper.java:193)
       at android.app.ActivityThread.main(ActivityThread.java:5345)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:515)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:824)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:640)
       at dalvik.system.NativeStart.main(NativeStart.java)
```

**Comment by aforcier** [comment]
It seems like this is a user who's skipped several versions of the app, upgrading from `6.3` or earlier directly to `7.0-rc-2` and skipping the migration step that added `dateLastUpdated`.

**Comment by maxme** [comment]
Yeah, I thought of this last week, db migration + fluxc migration = ☠️

**Comment by aforcier** [comment]
Fixed in #5454.
